src/components/AddressResolver/EndpointEntry/EndpointEntry.tsx

```diff
@@ -135,92 +136,134 @@ export const EndpointEntry: FunctionComponent<EndpointEntryProps> = ({
           entityLookup: features.entityLookup?.location,
         },
       });
+
+      showOverlay(<AddResolverConfirmation />);
     } catch (e) {
       onSetAllError(e.message);
     }
     setIsLoading(false);
   };
 
   return (
-    <tr className={className}>
-      <th>
+    <div
+      className={`flex flex-col bg-white rounded-xl shadow-lg w-80 mt-6 pt-12 pb-8 md:even:bg-cerulean-50 md:bg-none md:w-full md:h-auto md:rounded-none md:shadow-none md:mt-0 md:p-4 group ${
+        isEditable ? "h-130" : "h-auto"
+      }`}
+    >
+      <div className="flex flex-col md:flex-row">
+        <div
+          className={`flex flex-col text-2xl leading-3 mr-3.5 text-cloud-300 visible ${
+            !isEditable ? "md:invisible md:group-hover:visible" : "md:invisible"
+          }`}
```

Comment:
check the mobile layouts here:

![Screenshot 2021-07-27 at 9 14 57 AM](https://user-images.githubusercontent.com/4774314/127079543-61d35bfe-906e-4adc-a4af-ffe6998589bc.png)
![Screenshot 2021-07-27 at 9 15 28 AM](https://user-images.githubusercontent.com/4774314/127079545-4a3b33f3-d5c4-4c51-afc2-ee5110bb2500.png)

Reply:
done. i shift the arrows next to the trash icon for the time being as jerry told me to wait for him to come up with an actual design for it.